.env

```diff
@@ -0,0 +1,2 @@
+GITHUB_CLIENT_ID: 40215582fd61ce58b59b
+GITHUB_CLIENT_SECRET: 3900adb88d2eebecfccf8bf55ba3b69bfa44a79a
```

Comment:
Hey an Oauth Token!  Hmmm... Now I can make a site masquerading as yours!

I see you added `.env` to your `.gitignore` file, but you need to remove this from your git history to get it to stop showing up.